app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  extend Enumerize
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :trackable, :validatable
+
+  enumerize :role, in: [:student, :tutor, :admin]
```

Comment:
Why not to use built in rails `enum` instead of adding `enumerize` gem?
Also prefer to use hash for enum like:
`enum role: {0: :student, 1: :tutor, 2: :admin}.freeze`